Paco-Server/ear/default/web/partials/advanced.html

```diff
@@ -101,6 +101,7 @@
         <input type="text" ng-model="group.feedback.text" placeholder="Message">
       </md-input-container>
       <no-ios ng-if="group.feedback.type === 0 && group.feedback.text !== defaultFeedback">iOS does not support custom messages</no-ios>
+      <no-ios ng-if="group.feedback.type != 0">No iOS support</no-ios>
```

Comment:
Could you add a topic at the end, e.g.,"No iOS support _for non-static feedback"_